pulsar-client/src/test/java/org/apache/pulsar/client/impl/schema/generic/MultiVersionSchemaInfoProviderTest.java

```diff
@@ -61,7 +62,7 @@ public void testGetSchema() throws Exception {
                         any(TopicName.class),
                         any(byte[].class)))
                 .thenReturn(completableFuture);
-        SchemaInfo schemaInfoByVersion = schemaProvider.getSchemaByVersion(new byte[0]).get();
+        SchemaInfo schemaInfoByVersion = schemaProvider.getSchemaByVersion(new LongSchemaVersion(0).bytes()).get();
```

Comment:
it looks like we are changing the behaviour.
can this be a problem ?
do we need to add some compatibility layer ?

Reply:
The change of the class MultiVersionSchemaInfoProvider is not necessary and I revert this test. Thanks.